### PR TITLE
feat: update chapter 7 of beginner guide

### PR DESCRIPTION
**Pull request by Popespice**
## Summary
Updates chapter 7 of the BG.

### Location
docs/pilots-corner/a380x/a380x-beginner-guide/07_landing.md

Discord username (if different from GitHub): Popespice

### CODE REVIEW

**Review by Popespice — COMMENTED** [review]
First draft of chapter 7 updates. Still not sure why it thinks im updating chapter 3.

File: docs/pilots-corner/a380x/a380x-beginner-guide/.pages [hunk 1]
```diff
@@ -12,6 +12,6 @@ nav:
     - Engine Start and Taxi: 04_engine-start-taxi.md
     #- 05_takeoff-climb-cruise.md
     #- 06_descent.md
-    #- 07_landing.md
+    - 07_landing: 07_landing.md
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/.pages]: page title missing

**Review by frankkopp — COMMENTED** [review]
I have not yet reviewed this page but I have the same remark as on the other pages: 
Instead of the A32NX bg use the SOP as your template - e.g. you could even remove all the A320 stuff and build it up new (you can always copy stuff over)

File: docs/pilots-corner/a380x/a380x-beginner-guide/03_preparing-fms.md [hunk 2]
```diff
@@ -358,14 +360,14 @@ If we would like to have additional navaids for the departure, we can input the
 
 ^^VOR^^
 
-The A32NX supports VOR autotuning when in range of a VOR before departure. You can verify this by checking the RADNAV page and seeing if the VOR frequency is already populated. You should verify the relevance of this VOR to your departure procedure and flight plan before takeoff.
+The A380X supports VOR autotuning when in range of a VOR before departure. You can verify this by checking the RADNAV page and seeing if the VOR frequency is already populated. You should verify the relevance of this VOR to your departure procedure and flight plan before takeoff.
```
Comment (2hwk) [inline docs/pilots-corner/a380x/a380x-beginner-guide/03_preparing-fms.md]:
```suggestion
The A380X supports VOR autotuning when in range of a VOR before departure. You can verify this by checking the POSITION / NAVAIDS page accessed via the MFD or on the RMP, and seeing if the VOR frequency is already populated. You should verify the relevance of this VOR to your departure procedure and flight plan before takeoff.
```

**Pull request merged**